templates/bundles/DemosPlanCoreBundle/DemosPlanProcedure/administration_edit.html.twig

```diff
@@ -1277,13 +1315,20 @@
 
             {# form controls when not in wizard mode #}
             <div class="text-right space-inline-s">
+                {# A hidden input is used here to prevent the form from being submitted in wizard mode when the next button is clicked #}
                 <input
+                    type="submit"
+                    class="hidden"
+                    value="{{ "save"|trans }}">
```

Comment:
Why is the input needed?

Reply:
Oh, actually I can confirm that this input prevents the form to be submitted when in assistant or wizard mode, but I have absolutely no idea why. @sakutademos could you explain this to me? I really don't get it but it works =)

Reply:
to add some actions when the user clicks the save button, I had to change it from input `type="submit"` to button and add a `submit()` method in `DpBasicSettings`, where I included additional actions before submitting the form. However, as I understand it, the wizard mode requires `type="submit"` because, without it, every button is treated as a submit button. That is my assumption

Reply:
Thank you, this makes sense to me and I suggest we leave it like that since this does the trick.